resources/js/export.js

```diff
@@ -0,0 +1,410 @@
+import * as dayjs from "dayjs";
+import * as dayjsQuarterOfYear from "dayjs/plugin/quarterOfYear";
+import * as dayjsLocalizedFormat from "dayjs/plugin/localizedFormat";
+
+import { TransactionsApi } from "./transactions-api";
+
+dayjs.extend(dayjsQuarterOfYear);
+dayjs.extend(dayjsLocalizedFormat);
+
+const TransactionsExport = ({
+    address,
+    userCurrency,
+    rate,
+    network,
+    canBeExchanged,
+}) => {
+    const csvColumns = {
+        id: "TXID",
+        timestamp: "Timestamp",
+        sender: "Sender",
+        recipient: "Recipient",
+        amount: "Value [:networkCurrency]",
+        fee: "Fee [:networkCurrency]",
+        total: "Total [:networkCurrency]",
+        amountFiat: "Value [:userCurrency]",
+        feeFiat: "Fee [:userCurrency]",
+        totalFiat: "Total [:userCurrency]",
+        rate: "Rate [:userCurrency]",
+    };
+
+    const getTransactionAmount = (transaction) => {
+        let amount = transaction.amount / 1e8;
+        if (transaction.type === 6 && transaction.typeGroup === 1) {
```

Comment:
```suggestion
        if (transaction.type === 6 && transaction.typeGroup === 1) {
```

might be nice to change this to enums. They exist in PSDK or in the JS crypto package iirc